### ISSUE TITLE
When selecting a neume, associated bounding box should also be highlighted

### ISSUE THREAD

**Issue by thesoundbyle** [DDMAL/Neon#669]
Bounding boxes are not highlighted when a neume is selected, but when a bounding box a selected the paired neume is highlighted

![BBox selected with Neumes highlighted](https://user-images.githubusercontent.com/62864127/118163068-897a9580-b3ef-11eb-827b-0bf531389e4a.jpg)
![Only Neumes selected](https://user-images.githubusercontent.com/62864127/118163071-8a132c00-b3ef-11eb-9d12-2557be61719a.jpg)

**Comment by martha-thomae** [comment]
@thesoundbyle: Is this actually a bug or more like a feature request?

@JRegimbal: was there a reason why this was designed this way?

**Comment by annamorphism** [comment]
It highlights if you select as syllable, but not as a neume--the syllables have a closer 1-to-1 relationship with the textboxes. I see having the neume selection highlight the text as a feature request and not a bug. 
In any case, this behavior is also true in browser (1c34e14).

**Comment by yinanazhou** [comment]
Hi @JoyfulGen and @annamorphism, what do you think about this feature?

**Comment by annamorphism** [comment]
I see it as nice-to-have, if it's easy to implement. Thoughts, @JoyfulGen?

**Comment by JoyfulGen** [comment]
@yinanazhou I agree! This is really not urgent; as @annamorphism said, neumes aren't directly related to the text boxes, so I'd actually never even noticed this situation. But this feature can't hurt!

**Comment by JoyfulGen** [comment]
I disagree with myself: this behaviour makes perfect sense. A neume is its own entity, separate from the syllable it lives in, whereas a bounding box is intrinsically linked to a syllable. Saying that a neume is highlighted when a bounding box is selected is incorrect; the associated _syllable_ is highlighted, because bboxes and syllables are basically one and the same. Selecting one neume doesn't necessarily mean selecting a syllable, because a syllable can contain multiple neumes. So it makes perfect sense that when a syllable is selected, the bbox gets highlighted, but when a neume is selected, it does not. I'm closing the issue.

**Comment by annamorphism** [comment]
eh, I understand the feature request as "show which bit of text is this neume associated with" (since the neume is contained in some syllable) for people who don't feel like toggling between neume and syllable modes with @JoyfulGen-like speed. but i also don't think it's that useful a feature (and stems from an older version of Neon, at this point).
